test/quicksiler-config-client.test.js

```diff
@@ -18,4 +18,25 @@ describe('QuicksilverConfigClient', () => {
       mbConfigPath: process.env.MB_CONFIG_PATH,
     });
   }
+
+
+  // Constructor.
+  describe('constructor', () => {
+    // Test new instance.
+    it('without options should throw a TypeError', () => {
+      (() => new QuicksilverConfigClient()).should.throw(TypeError);
+    });
+
+    // Check API base URL.
+    it('base URL option should be set', () => {
+      process.env.MB_CONFIG_PATH.should.be.not.empty();
```

Comment:
I think it's better to use more relevant name for the env variable, so it's easier to manage in future. How about `QS_CONFIG_PATH`?